tests/span/test_event.py

```diff
@@ -1,36 +1,63 @@
 # (c) Copyright IBM Corp. 2024
 
 import time
+from typing import Generator
 
-from instana.span.readable_span import Event
-
-
-def test_span_event_defaults():
-    event_name = "test-span-event"
-    event = Event(event_name)
+import pytest
 
-    assert event
-    assert isinstance(event, Event)
-    assert event.name == event_name
-    assert not event.attributes
-    assert isinstance(event.timestamp, int)
-
-
-def test_span_event():
-    event_name = "test-span-event"
-    attributes = {
-        "field1": 1,
-        "field2": "two",
-    }
-    timestamp = time.time_ns()
+from instana.span.readable_span import Event
 
-    event = Event(event_name, attributes, timestamp)
 
-    assert event
-    assert isinstance(event, Event)
-    assert event.name == event_name
-    assert event.attributes
-    assert len(event.attributes) == 2
-    assert "field1" in event.attributes.keys()
-    assert "two" == event.attributes.get("field2")
-    assert event.timestamp == timestamp
+class TestEvent:
+    @pytest.fixture(autouse=True)
+    def _resource(
+        self,
+    ) -> Generator[None, None, None]:
+        self.event = None
+        yield
```

Comment:
The same as before. If this is the only change in this file, I don't see any reason to create a class for the whole test. One of the advantages of using pytest is that it is unnecessary to create a test class to run tests.